dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/RestMetricsTest.java

```diff
@@ -223,6 +225,7 @@ public void testCheckedExceptions() throws IOException {
         runCheckMonitorStats(200, CHECKED_METHOD_INDEX);
     }
 
+    @SkipForRepeat("EE10_FEATURES") //Skipping until MP Metrics is upgraded for MP 6.0
```

Comment:
and this one

Reply:
https://github.com/OpenLiberty/open-liberty/issues/22519